AppCenterAnalytics/AppCenterAnalytics/Internals/MSAnalyticsPrivate.h

```diff
@@ -10,6 +10,9 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+// The Id for critical events.
+static NSString *const kMSCriticalChannelPostfix = @"critical";
```

Comment:
I noticed that another variable used later uses "suffix": `kMSOneCollectorGroupIdSuffix`

We should align this with preexisting names and use `kMSCriticalChannelSuffix`